biogrid/libs/biogrid-simulator/src/lib/bioenergy-source/solar-panel.spec.ts

```diff
@@ -64,7 +64,8 @@ describe('tests for the BioEnergySource', () => {
     );
     const weather = new WeatherLib(date, longitude, latitude);
     await weather.setup();
-    date.setHours(date.getHours() + 10);
-    expect(await energySource.getPowerAmount(date)).toEqual(1.212440625);
+    // 10 AM
+    date.setHours(10);
+    expect(await energySource.getPowerAmount(date)).toBeGreaterThan(0);
```

Comment:
what happens if it happened to be cloudy that day? another option we could do is create a mock object for the API and just test that the API does in fact get called. Just an option - let me know what you think

Reply:
Ok sure, sounds good. I think even on a cloudy day though there is still some energy being produced

Reply:
Is this 10AM local time, or GMT?  Because 10AM GMT is the middle of the night elsewhere

Reply:
10 AM local, will comments